validation/1.4/modules/ROOT/pages/index.adoc

```diff
@@ -1,256 +1,38 @@
-= Validation Module - Mule 4
+= Validation Module 1.4 - Mule 4
 :page-aliases: connectors::validation/validation-connector.adoc
 
 Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
 
-Validation Module v1.4
+The Validation module verifies that the content of a message in a Mule flow matches specific criteria. The module provides explicit messages about the cause of an exception in a flow. Based on the specific data you are validating, you can customize the exception message that is displayed in the logs.
 
-The Validation module can verify that the contents of a message in a Mule flow match specific criteria. The module provides explicit messages about the cause of an exception in a flow. Based on the specific data you are validating, you can customize the exception message that is displayed in the logs.
+If a message does not meet the defined validation criteria, the validation fails and returns a validation error, such as `VALIDATION:INVALID_URL`, which is a `ValidationException`. By default, this message contains customizable text.
```

Comment:
This clarifies, but it also could be said more simply earlier, so that you need not clarify in subsequent text like this. The second para repeats the first, as follows:

Note:
Be sure to assign one term to one thing. The text so far uses these slightly different identifiers:

- module verifies that the content of a message matches specific criteria
VS
 if message (content of a message) does not match the defined validation criteria (specific criteria)

- module provides explicit messages about the cause of an exception
VS
 the validation fails (exception) and returns a validation error (provides explicit messages)

- you can customize the exception message 
VS
 this message contains customizable text